tests/system-test/3-connectors/c#/TDengineDriver/TDengineDriver.cs

```diff
@@ -142,6 +143,7 @@ public struct TAOS_MULTI_BIND
 
         // line number, or the values number in buffer 
         public int num;
+        public bool is_string;
```

Comment:
The struct need to keep as same as taosc's  struct. Cannot add new field "is_string"